settings/src/style.scss

```diff
@@ -1,6 +1,10 @@
 @import '~@wordpress/base-styles/colors';
 @import '~@wordpress/base-styles/colors.native';
 
+#main {
+	margin-bottom: 2rem;
+}
```

Comment:
I _think_ this makes sense. Just double check that it will work in your sandbox once this code is in place.